Dockerfile.dev

```diff
@@ -0,0 +1,56 @@
+FROM node:10 as frontend-builder
+
+WORKDIR /frontend
+COPY package.json package-lock.json /frontend/
+RUN npm install
+
+COPY client /frontend/client
+COPY webpack.config.js /frontend/
+RUN npm run build
+
```

Comment:
Now that we have a dedicated Docker file for development we no longer need this here, as we build frontend assets on the host.

Reply:
By deleted this build step, then the some tests which depend on client/dist/ became to fail. So I changed index handler to return 200 with error.html in that TemplateNotFound case https://github.com/getredash/redash/pull/4292/commits/6ab16ecb503dda35294c91c6dd4f5a69703b0695